packages/docs/pages/self-host/config.mdx

```diff
@@ -75,6 +87,15 @@ Optional. If not specified, GrowthBook will use the local file system to store u
 - **AWS_ACCESS_KEY_ID** (not required when deployed to AWS with an instance role)
 - **AWS_SECRET_ACCESS_KEY** (not required when deployed to AWS with an instance role)
 
+#### Google Cloud Services File Uploads
+
+Optional. If specified, GrowthBook will use your Google Cloud Services instance to store uploads.
+
+- **UPLOAD_METHOD** (set to "google-cloud" instead of default value "local")
+- **GCS_PRIVATE_KEY_FILEPATH** (path to service account key )
```

Comment:
@jdorn Do you think it's worth adding any guidance here on how this env variable isn't required if GrowthBook is deployed to GCS?

Reply:
This is the wrong env variable name, isn't it?  Should be `GOOGLE_APPLICATION_CREDENTIALS`.  And yes, we should add a similar note to the `AWS_ACCESS_KEY_ID` above.  I don't think we need to say it's a path to the service account key.  People using google cloud should know that already.

Reply:
Updated! The name was indeed outdated.